content/docs/multi-user/overview.md

```diff
@@ -0,0 +1,116 @@
++++
+title = "Overview of Multi-user Isolation Support"
+description = "Isolation of user created resources for convenience and organization"
+weight = 5
++++
+
+Kubeflow v0.6 release enables support for multi-user isolation of user created
+resources in a Kubeflow deployment. The primary purpose of this functionality
+is to enable multiple users to operate on a shared Kubeflow deployment without
+stepping on each others' jobs and resources. It provides the users with the
+convenience of clutter free browsing of notebooks, training jobs, serving
+deployments and other resources. The isolation mechanisms also prevent
+accidental deletion/modification of resources of other users in the deployment.
+Note that the isolation support in Kubeflow doesn't provide any hard security
+guarantees against malicious attempts by users to infiltrate other user's
+profiles.
+
+## Usage overview
+
+An administrator needs to deploy Kubeflow and configure the authentication.
+service for the deployment.  A user can log into the system and will by default
+be accessing their primary profile. Users have view and modify access to their
+primary profile. The owner of a profile can share access to the profile with
+another user in the system. When sharing the access to a profile, the owner can
+choose to share it as a view access only or view and modify access with the
```

Comment:
`view or edit`?

Reply:
I had discussed with Jeremy and agreed that we'll use view and modify and keep that terminology consistent.